### ISSUE TITLE
Missing dependency

### ISSUE THREAD

**Issue by jvoorhis** [fabiolb/fabio#473]
Fabio depends on magiconair/go-metrics-statsd, but the upstream repository has been removed. A copy of the repo currently resides in vendor, so builds are not affected.

Are there any plans to restore the repo, or switch to an alternate implementation?

**Comment by magiconair** [comment]
This was just a monkey patch until this was fixed upstream. I’ll update fabio to use the pubnub repo again. 

—
Frank Schröder

**Comment by marco-m-pix4d** [comment]
Hello @tristanmorgan, the vendor directory doesn't exist any more and go.mod doesn't contain magiconair/go-metrics-statsd, so it seems that this ticket can be closed ?

**Comment by tristanmorgan** [comment]
@marco-m-pix4d good catch. Yes we migrated to go modules a while back so this issue can be closed
